ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: cucushift-installer-custom-cidr
+  from_image:
```

Comment:
I think this step use the same image as `ovn-conf` ref.

Reply:
updated, thanks